static/app/views/performance/onboarding.tsx

```diff
@@ -157,7 +167,10 @@ function Onboarding({organization, project}: Props) {
         onClick={event => {
           event.preventDefault();
           window.location.hash = 'performance-sidequest';
-          SidebarPanelStore.activatePanel(SidebarPanelKey.PerformanceOnboarding);
+          dispatchSidebar({
```

Comment:
I kind of wish what we got back from `useSidebar` was like `{activePanel, activatePanel, deactivatePanel}`

Reply:
That is an option, but I dont think that having an intermediary method is super high value - we'd still have to map to reducer actions and then we'd also need to ensure we have stable references which bring in the whole useCallback etc. I also like to keep access to dispatch directly to make it easily extendable